### PR TITLE
Edit IBM Cloud-Kubeflow: Securing the Kubeflow Authentication with HTTPS using the network load balancer

### PR DESCRIPTION
**Pull request by 8bitmp3**
Hey @shawnzhu I took the guide through a linter and refactored some text and reformatted some parts. (Hopefully, soon we'll get a Markdown linter working during testing.)

Note that in `Exposing the Kubeflow dashboard with DNS and TLS termination`, there is a step between 5 and 6:

```
Update the `istio-ingressgateway-certs.yaml` file by changing the value of 
`metadata.name` to `istio-ingressgateway-certs`
```

Please double check that it should be number 6, in which case there are 7 steps in total.

I've also addressed some accessibility stuff, since we shouldn't use "see... guide", if we can (-> "go to").

### CODE REVIEW

File: content/en/docs/ibm/deploy/authentication.md [hunk 1]
```diff
@@ -85,13 +136,24 @@ These instructions are using Kubeflow dashboard as an example, however, they app
         privateKey: /etc/istio/ingressgateway-certs/tls.key
         serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
     ```
-7. Verify that the traffic is routed via HTTPS by using the value of above `Hostname` in your browser. It should redirect traffic from an HTTP address to HTTPS address automatically.
 
-**Note**: The certificates for the NLB DNS host secret expire every **90** days. The secret in the `istio-system` namespace is automatically renewed by IBM Cloud Kubernetes Service 37 days before it expires. After this secret is updated, you must manually copy it to the `istio-ingressgateway-certs` secret by repeating commands in step 6.
+7. Verify that the traffic is routed via HTTPS by using the value of
```
Comment (shawnzhu) [inline content/en/docs/ibm/deploy/authentication.md:140]: this should be step 8.

**Review by shawnzhu — COMMENTED** [review]
@8bitmp3 Thanks for the PR. I'm ok with these changes. it's just those line feeds make code review hard. do you use a fixed width to format markdown?

**Comment by shawnzhu** [comment]
@8bitmp3 I can merge this as long as those additional line feeds are removed.

File: content/en/docs/ibm/deploy/authentication.md (hunk 1 above)
Comment (8bitmp3) [reply content/en/docs/ibm/deploy/authentication.md:140]: Thanks for spotting this. Can you fix it in your repo please so it displays "8. Verify..."?

> it's just those line feeds make code review hard. do you use a fixed width to format markdown?

Wrapping at (or around) 80 characters per line is a common practice (we can treat this as docs as code). We should ideally implement this in a doc test, as some lines can become infinitely long and, therefore, also hard to review.

Comment (shawnzhu) [reply content/en/docs/ibm/deploy/authentication.md:140]: > Wrapping at (or around) 80 characters per line is a common practice (we can treat this as docs as code). We should ideally implement this in a doc test, as some lines can become infinitely long and, therefore, also hard to review.

It totally makes sense when we identify this out of this PR. it's just inconsistent with the rest of Markdown in this repo. I would suggest making this rule explicit and possibly enforce via a future linter, then I'm happy to accept it, or I would have to cherry-pick your commit later.

**Comment by shawnzhu** [comment]
cherry-picked commit into https://github.com/kubeflow/website/pull/2071

**Pull request closed without merging**